game.js

```diff
@@ -1,51 +1,68 @@
-const rls = require('readline-sync')
+const rls = require("readline-sync");
 
 /**
  * Starts the game by prompting the user if they want to play
  * Calls either gameLoop() or quitGame()
- * 
+ *
  * @returns {undefined}
  */
-const startGame = () => {
 
+
+const gameLoop = () =>{
+console.log("Let's start!")
+console.log("I have a random number in mind")
+console.log("It's between 1 and 1000")
+console.log("You have 10 guesses total")
+}
+
+const quitGame = () => {
+  console.log("Have a nice life!")
+  console.log("Goodbye!")
+  process.exit()
+}
+
+const startGame = () => {
```

Comment:
I like how you reordered your functions instead of just changing to ES5 in order to call them later in your startGame function